### PR TITLE
Support media queries

### PR DESCRIPTION
**Pull request by rolfen**
Hello,
 - Media Query are supported (with test)
 - Selector indentation is preserved in output
 - ava updated to current version

### CODE REVIEW

File: tests/fixtures/nested-input.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        offset-inline-start: 1px;    
+    }
+
+    @media screen and (min-width: 768px) {
+
+        .bar{
+            offset-inline-end: 1px;    
+        }
+    }
+}
```
Comment (gasolin) [inline tests/fixtures/nested-input.css]: please add a blank line at end of file

File: tests/fixtures/nested-output.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+        html[dir="ltr"] .bar{
+            right: 1px;    
+        }
+
+        html[dir="rtl"] .bar{
+            left: 1px;    
+        }
+    }
+}
```
Comment (gasolin) [inline tests/fixtures/nested-output.css]: please add a blank line at end of file

File: index.js [hunk 3]
```diff
@@ -198,39 +200,42 @@ function postcssBiDirection(opts) {
...
-                // multiple selectors in a rule
-                let selectors = item.ltrRule.selector.split(PATTERN);
-                if (selectors.length) {
-                    item.ltrRule.selector =
-                      selectors.join(',\nhtml[dir="ltr"] ');
-                }
+                // prefix each comma-separated selector
+                item.ltrRule.selector = item.ltrRule.selector
+                    .split(PATTERN)
+                    .map(function(selector, i){
```
Comment (gasolin) [inline index.js]: add a space between function params and { `(selector, i) {` for lint

File: index.js [hunk 4]
```diff
@@ -186,6 +186,8 @@ function postcssBiDirection(opts) {
         // Transform CSS AST here
         // Unefficient but works
 
+
```
Comment (gasolin) [inline index.js]: can remove these extra blank line

File: package.json [hunk 5]
```diff
@@ -21,7 +21,7 @@
     "postcss": "^5.0.10"
   },
   "devDependencies": {
-    "ava": "^0.4.2",
+    "ava": "^0.23.0",
```
Comment (gasolin) [inline package.json]: thanks!

**Comment by rolfen** [comment]
Hi,
I think you can choose to squash into a single commit, when you merge.
I believe that you should be able to add commits to my branch yourself, as I picked the option to "Allow edits from maintainers". I am generally not fussy about whitespace myself and it should be easy to revert the ava commit. Feel free to do so and let me know if you have any trouble.
Thanks

**Comment by rolfen** [comment]
Just to be clear: I don't want to go back and fix spaces.
Sorry for any inconvenience, and thanks for the great work on postcss-bidirection.

**Comment by gasolin** [comment]
It's my first time trying the `Edits from maintainers` feature, seems goes well, thanks!

**Comment by rolfen** [comment]
Thank you for fixing the whitespace un the source!

**Pull request merged**